mk/build.mk

```diff
@@ -105,6 +109,60 @@ else
 	@echo "CoreDNS is already built. If you want to rebuild it, remove the binary: rm $(BUILD_ARTIFACTS_DIR)/coredns/coredns"
 endif
 
+foo:
```

Comment:
should this be here?

Reply:
definitely not, thank you for catching this!

Reply:
addressed